tasmota/language/bg-BG.h

```diff
@@ -83,7 +83,7 @@
 #define D_DNS_SERVER "DNS Сървър"
 #define D_DONE "Изпълнено"
 #define D_DST_TIME "Лятно време"
-#define D_ECO2 "eCO₂"
+#define D_ECO2 "eCO2"
```

Comment:
Why change this?
CO₂ is the chimical notation for the Carbon dioxyd even in [bulgarian](https://bg.wikipedia.org/wiki/%D0%92%D1%8A%D0%B3%D0%BB%D0%B5%D1%80%D0%BE%D0%B4%D0%B5%D0%BD_%D0%B4%D0%B8%D0%BE%D0%BA%D1%81%D0%B8%D0%B4)

Reply:
I noticed this, but the editor didn't want to display the subscript (it appeared as a box), although the superscripts (like in m²) looked fine. So I though that this character is broken, looked at some other translation files and most of them had D_ECO2 spelled as "eCO2".

Now I see in the diff that the subscript looks good.

Reply:
Maybe you should setup you editor to UTF8 ;-)

Reply:
It uses UTF-8 already, otherwise I wouldn't be able to see the Cyrillic letters at all :)

Here's how I "see" the subscript character again:

![image](https://user-images.githubusercontent.com/20048364/70724937-6d9f2b80-1cf3-11ea-8339-dda92de9a2a5.png)

I'll have this in mind and I'll probably use another editor for future translations.

Reply:
VSCode and Notepad++ work well for that.
The funniest in that story is that the Bulgarian language was the only one that spelled it properly using subscript. Didn't notice that before I reviewed your changes.

Reply:
It's funny indeed, isn't it!

Now I remembered - the initial translation I did in VSCode, when I had some code changes too in my fork. But now I'm just tracking this repo, don't need my local changes anymore and so don't have the need to run "full IDE" like VSCode. That's the reason I use Notepad++ when catching up with translations. But, lessons learned - double check the diff before submitting the changes ;)